test/distributed/org/apache/cassandra/fuzz/ring/ConsistentBootstrapTest.java

```diff
@@ -177,39 +182,36 @@ public void coordinatorIsBehindTest() throws Throwable
                        .drop()
                        .on();
 
-                IInstanceConfig config = cluster.newInstanceConfig()
-                                                .set("auto_bootstrap", true)
-                                                .set(Constants.KEY_DTEST_FULL_STARTUP, true)
-                                                .set("progress_barrier_default_consistency_level", "NODE_LOCAL");
-                IInvokableInstance newInstance = cluster.bootstrap(config);
-                // Prime the CMS node to pause before the finish join event is committed
-                long[] metricCounts = new long[4];
-                for (int i = 1; i <= 4; i++)
-                    metricCounts[i - 1] = cluster.get(i).callOnInstance(() -> TCMMetrics.instance.coordinatorBehindPlacements.getCount());
-
                 DataTracker tracker = new DataTracker.SequentialDataTracker();
                 RingAwareInJvmDTestVisitExecutor executor = RingAwareInJvmDTestVisitExecutor.builder()
                                                                                             .replicationFactor(new TokenPlacementModel.SimpleReplicationFactor(2))
                                                                                             .nodeSelector(i -> 2)
                                                                                             .consistencyLevel(ConsistencyLevel.ALL)
+                                                                                            .retryPolicy(InJvmDTestVisitExecutor.RetryPolicy.NO_RETRY)
                                                                                             .build(schema,
                                                                                                    tracker,
                                                                                                    new QuiescentChecker(schema.valueGenerators, tracker, history),
                                                                                                    cluster);
 
+                // Prime the CMS node to pause before the finish join event is committed
+                Callable<?> pending = pauseBeforeCommit(cmsInstance, (e) -> e instanceof PrepareJoin.MidJoin);
```

Comment:
so block midjoin *before* we create/start the node rather than after?  makes sense